File: src/main/java/ddp/api/DataCacheConfiguration.java
package ddp.api;

public class DataCacheConfiguration
{
    private String _hostname;
    private String _ipAddress;
    private int _port;
    private String _cacheName;

    public DataCacheConfiguration(String hostname, String ipAddress, int port, String cacheName)
    {
        _hostname = hostname;
        _ipAddress = ipAddress;
        _port = port;
        _cacheName = cacheName;
    }

    public String getHostname()
    {
        return _hostname;
    }

    public String getIpAddress()
    {
        return _ipAddress;
    }

    public int getPort()
    {
        return _port;
    }

    public String getCacheName()
    {
        return _cacheName;
    }

    @Override
    public String toString()
    {
        StringBuilder stringBuilder = new StringBuilder();
        stringBuilder.append("DataCacheConfiguration{ ");
        stringBuilder.append("hostname= ").append(getHostname());
        stringBuilder.append(", ");
        stringBuilder.append("ipAddress= ").append(getIpAddress());
        stringBuilder.append(", ");
        stringBuilder.append("port= ").append(getPort());
        stringBuilder.append(", ");
        stringBuilder.append("cacheName= ").append(getCacheName());
        stringBuilder.append(" }");

        return stringBuilder.toString();
    }
}

File: src/main/csharp/Ddp.Api/DataCacheConfiguration.cs

namespace Ddp.Api
{
    public class DataCacheConfiguration
    {
        private string _hostname;
        private string _ipAddress;
        private int _port;
        private string _cacheName;

        public DataCacheConfiguration(string hostname, string ipAddress, int port, string cacheName)
        {
            _hostname = hostname;
            _ipAddress = ipAddress;
            _port = port;
            _cacheName = cacheName;
        }

        public string Hostname
        {
            get { return _hostname; }
        }

        public string IpAddress
        {
            get { return _ipAddress; }
        }

        public int Port
        {
            get { return _port; }
        }

        public string CacheName
        {
            get { return _cacheName; }
        }

        public override string ToString()
        {
            string toString = "DataCacheConfiguration{ hostname= " + Hostname
                + ", ipAddress= " + IpAddress
                + ", port= " + Port
                + ", cacheName= " + CacheName
                + " }";

            return toString;
        }
    }
}